Utilities/WearableJSONExporter/marketplace-packager.js

```diff
@@ -65,7 +65,8 @@
 
     var exportProperties = {
         type: "Model",
-        parentID: MyAvatar.sessionUUID,
+        parentID: "{00000000-0000-0000-0000-000000000001}",
+        owningAvatarID: "{00000000-0000-0000-0000-000000000000}",
```

Comment:
replace with Uuid.NULL

Reply:
NOTE: owningAvatarID should not be settable on creation, if it can that is a bug. It's a read-only property to see the current avatar that the avatar-entity belongs to.